src/contributors/MapContributor.ts

```diff
@@ -2318,6 +2342,11 @@ export class MapContributor extends Contributor {
                     topologyLayer.providedFields = topologyLayer.providedFields ?
                         existingTopologyLayer.providedFields.concat(topologyLayer.providedFields) : existingTopologyLayer.providedFields;
                 }
+                if (existingTopologyLayer.includeFields) {
+                    topologyLayer.includeFields = topologyLayer.includeFields ?
+                        new Set([...existingTopologyLayer.includeFields].concat([...topologyLayer.includeFields])) :
```

Comment:
I don't think you need to concat, but `...existingTopologyLayer.includeFields, ...topologyLayer.includeFields)`